packages/main/src/CalendarHeader.js

```diff
@@ -97,11 +112,19 @@ class CalendarHeader extends UI5Element {
 		const yearFormat = DateFormat.getDateInstance({ format: "y", calendarType: this.primaryCalendarType });
 		const localDate = new Date(this.timestamp * 1000);
 		const calendarDate = CalendarDate.fromTimestamp(localDate.getTime(), this.primaryCalendarType);
-
 		this._monthButtonText = localeData.getMonths("wide", this.primaryCalendarType)[calendarDate.getMonth()];
 		this._yearButtonText = yearFormat.format(localDate, true);
 		this._prevButtonText = this.i18nBundle.getText(CALENDAR_HEADER_PREVIOUS_BUTTON);
 		this._nextButtonText = this.i18nBundle.getText(CALENDAR_HEADER_NEXT_BUTTON);
+
+		if (this.hasSecondaryCalendarType) {
```

Comment:
In a meeting with the web components team we got to a conclusion that all calculations with dates
logically should happen at Calendar component level and not in the CalendarHeader component.
The CalendarHeader component doesn't need to understand dates and later on will be incorporated in the Calendar component itself.